test/Performance/DynamoPerformanceTests/app.config

```diff
@@ -0,0 +1,63 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <runtime>
+    <assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1">
+      <dependentAssembly>
+        <assemblyIdentity name="System.Runtime.InteropServices.RuntimeInformation" publicKeyToken="b03f5f7f11d50a3a" culture="neutral" />
+        <bindingRedirect oldVersion="0.0.0.0-4.0.2.0" newVersion="4.0.2.0" />
```

Comment:
did Nuget add these? - I wonder if this has to do with downgrading...with the newer version already installed. Unclear if we actually want these statements.

Reply:
This file was autogenerated. I didn't give it much thought.